src/server/pageRendering.js

```diff
@@ -95,12 +97,19 @@ const getPlaceholder = placeholders => {
 	return placeholder;
 };
 
+const FILE_PATH = path.resolve('theme/assets/index.html');
 const renderPage = (req, res, store, themeText, placeholders) => {
 	const appHtml = getAppHtml(store, req.url);
 	const state = store.getState();
 	const head = getHead();
 	const placeholder = getPlaceholder(placeholders);
 
+	fs.readFile(FILE_PATH, 'utf8', (err, data) => {
```

Comment:
add one time read into a variable so it will server from there for next calls. when it is null then re-initialized the variable from index.html

will significantly increases the response time

Reply:
oke you're right implemented with singleton

Reply:
Thanks! There is your modified file.

https://github.com/Cezerin2/cezerin2-store/pull/19/commits/cc79ee47fbddd78dc34a8472ad99f843576472db